src/components/Pagination/Pagination.jsx

```diff
@@ -2,7 +2,7 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import classNames from 'classnames';
-import range from 'lodash/range';
+import { range } from 'lodash';
```

Comment:
Does this make a difference? I don't have a personal preference - happy to go with whichever has the best performance.

Reply:
Shouldn't make a difference as far as I can tell. Only changed this to make all the imports consistent.